Lib/test/test_cmd_line_script.py

```diff
@@ -810,6 +810,12 @@ def test_script_as_dev_fd(self):
                 out, err = p.communicate()
                 self.assertEqual(out, b"12345678912345678912345\n")
 
+    def test_filter_syntax_warnings_by_module(self):
+        filename = support.findfile('test_import/data/syntax_warnings.py')
+        rc, out, err = assert_python_ok('-Werror', '-Walways:::test.test_import.data.syntax_warnings', filename)
+        self.assertEqual(err.count(b': SyntaxWarning: '), 6)
+        rc, out, err = assert_python_ok('-Werror', '-Walways:::syntax_warnings', filename)
+        self.assertEqual(err.count(b': SyntaxWarning: '), 6)
```

Comment:
You may add newlines for readability:

```suggestion
        filename = support.findfile('test_import/data/syntax_warnings.py')

        rc, out, err = assert_python_ok('-Werror', '-Walways:::test.test_import.data.syntax_warnings', filename)
        self.assertEqual(err.count(b': SyntaxWarning: '), 6)

        rc, out, err = assert_python_ok('-Werror', '-Walways:::syntax_warnings', filename)
        self.assertEqual(err.count(b': SyntaxWarning: '), 6)
```